files/amazon-2/sge-tcsh.patch

```diff
@@ -0,0 +1,20 @@
+patch e91cde95c7e32f6ff6d8bdad463189c1da296cf4
```

Comment:
if this diff it's the same used for the centos-8 case, cannot we put this under the `default` folder and use it for both alinux2 and centos8?